src/wta/activity_transitions.py

```diff
@@ -171,92 +186,36 @@ def __mark_activity_transitions(
                 non_concurrent_previous_event_found = True
 
 
-def __calculate_frequency_and_duration(transitions: pd.DataFrame,
-                                       log_ids: Optional[EventLogIDs] = None) -> pd.DataFrame:
-    log_ids = log_ids_non_nil(log_ids)
-
-    # calculating frequency per case of the transitions with the same activities and resources
-    columns = transitions.columns.tolist()
-    transition_with_frequency = pd.DataFrame(columns=columns)
-    for (pair, records) in transitions.groupby(by=['source_activity', 'source_resource',
-                                                   'destination_activity', 'destination_resource']):
-        transition_with_frequency = pd.concat([transition_with_frequency, pd.DataFrame({
-            'source_activity': [pair[0]],
-            'source_resource': [pair[1]],
-            'destination_activity': [pair[2]],
-            'destination_resource': [pair[3]],
-            'frequency': [len(records)],
-            'transition_type': [records['transition_type'].iloc[0]],
-            log_ids.wt_total: [records[log_ids.wt_total].sum()],
-            log_ids.wt_batching: [records[log_ids.wt_batching].sum()],
-            log_ids.wt_prioritization: [records[log_ids.wt_prioritization].sum()],
-            log_ids.wt_contention: [records[log_ids.wt_contention].sum()],
-            log_ids.wt_unavailability: [records[log_ids.wt_unavailability].sum()],
-            log_ids.wt_extraneous: [records[log_ids.wt_extraneous].sum()],
-        })], ignore_index=True)
-    return transition_with_frequency
-
-
-def __create_single_dataframe(items: List[pd.DataFrame]) -> Optional[pd.DataFrame]:
-    items = list(filter(lambda df: not df.empty, items))
-    if len(items) == 0:
+def time_to_seconds(t):
```

Comment:
Please, use type hints. They really help to understand faster what a function expects as input.